eZ/Publish/API/Repository/Values/Content/DraftList/Item/ContentDraftListItem.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\API\Repository\Values\Content\DraftList\Item;
+
+use eZ\Publish\API\Repository\Values\Content\DraftList\ContentDraftListItemInterface;
+use eZ\Publish\API\Repository\Values\Content\VersionInfo;
+
+/**
+ * Item of content drafts list.
+ */
+class ContentDraftListItem implements ContentDraftListItemInterface
+{
+    /**
+     * @var \eZ\Publish\API\Repository\Values\Content\VersionInfo
```

Comment:
I thought you wanted one line class member docs... :trollface:

Reply:
It is not about what I want :)